client/src/app/site/pages/meetings/pages/projectors/modules/projector-detail/components/projector-detail/projector-detail.component.ts

```diff
@@ -368,4 +381,22 @@ export class ProjectorDetailComponent extends BaseMeetingComponent implements On
             })
         );
     }
+
+    public scrollPDFPage(direction: ScrollScaleDirection): void {
+        if (this.projector.scroll <= 0) {
+            this.summedOverflowSteps = 0;
+        }
+        direction === ScrollScaleDirection.Up
+            ? (this.summedOverflowSteps += this.projector.height / 100 - Math.floor(this.projector.height / 100))
+            : (this.summedOverflowSteps -= this.projector.height / 100 - Math.floor(this.projector.height / 100));
```

Comment:
```suggestion
        this._summedOverflowSteps += (this.projector.height % 100) / 100 * (direction === ScrollScaleDirection.Up ? 1 : -1);
```
I believe this would be equivalent.